pallets/staking/src/lib.rs

```diff
@@ -2413,13 +2435,7 @@ decl_module! {
         /// - Read Each: Bonded, Ledger, Payee, Locks, System Account (5 items)
         /// - Write Each: System Account, Locks, Ledger (3 items)
         /// # </weight>
-        #[weight =
-            ((120 * WEIGHT_PER_MICROS
-            + 54 * WEIGHT_PER_MICROS * Weight::from(T::MaxNominatorRewardedPerValidator::get())
-            + T::DbWeight::get().reads(7)
-            + T::DbWeight::get().reads(5)  * Weight::from(T::MaxNominatorRewardedPerValidator::get() + 1)
-            + T::DbWeight::get().writes(3) * Weight::from(T::MaxNominatorRewardedPerValidator::get() + 1)) * 25) / 100
-        ]
+        #[weight = weight::weight_for_payout_stakers::<T>()]
         pub fn payout_stakers(origin, validator_stash: T::AccountId, era: EraIndex) -> DispatchResult {
```

Comment:
Should we make `payout_stakers` uncallable? Or perhaps turn it into something that can only be called directly via governance (i.e. origin(root))?

Reply:
I keep it as it is because there may be some cases where the schedule slot may be missed for some unknown reasons then the user can easily withdraw the rewards for that era from the staking UI (polkodot for now). We can change it to governance but I think it will be overhead for the committee to monitor scheduling failure for the rewards.